### PR TITLE
Rule updated for es5 config regarding to issue #3

### PR DESCRIPTION
**Pull request by bcinarli**
The `no-extra-parens` rule updated as

``` js
'no-extra-parens': [ 'error', 'all', { ignoreJSX: 'multi-line', nestedBinaryExpressions: false }],
```

and `radix` rule updated as

``` js
'radix': ['error', 'as-needed'],
```

### CODE REVIEW

File: packages/eslint-config-adidas-es5/index.js [hunk 1]
```diff
@@ -258,7 +258,7 @@ module.exports = {
     ],
     'quote-props': [ 'error', 'consistent-as-needed' ],
     'quotes': [ 'error', 'single' ],
-    'radix': 'error',
+    'radix': ['error', 'as-needed'],
```
Comment (RecuencoJones) [inline packages/eslint-config-adidas-es5/index.js]: Pad array for consistency

**Review by RecuencoJones — CHANGES_REQUESTED** [review]

**Comment by moelders** [comment]
Fix the open issue and it LGTM.

File: packages/eslint-config-adidas-es5/index.js (hunk 1 above)
Comment (bcinarli) [reply packages/eslint-config-adidas-es5/index.js]: Done!

**Review by moelders — APPROVED** [review]
LGTM

**Review by RecuencoJones — APPROVED** [review]
LGTM! 🚀

**Pull request merged**